### PR TITLE
Fix Issue #10 MacOS build Break

### PR DESCRIPTION
**Pull request by Gansito144**
This also run deployment.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,19 @@
+# Golang version
 FROM golang:1.8
 
 MAINTAINER rafaelrendonpablo@gmail.com
 
+# Get the dependencies
 RUN go get github.com/onsi/ginkgo/ginkgo &&  go get github.com/onsi/gomega
 
+# Get & Build API
```
Comment (rendon) [inline Dockerfile]: I think these instructions should go after setting the working directory.

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+RUN go get github.com/algoristas/api
+RUN env GOOS=linux GOARCH=amd64 go build -o bin/api github.com/algoristas/api
+
+# Create application folder
 RUN mkdir -p /app
 
+# Set working directory
```
Comment (rendon) [inline Dockerfile]: I'm not sure, but seems like I forgot to move the the working directory (`cd /app`), could you verify please?

**Comment by Gansito144** [comment]
@rendon Comments were attended please provide comments about this new version.

**Comment by Gansito144** [comment]
Test Slack integration.

**Pull request merged**